controllers/storageoscluster/node-manager_operand.go

```diff
@@ -314,8 +315,8 @@ func getNodeManagerBuilder(fs filesys.FileSystem, obj client.Object, kcl kubectl
 	nextIndex := 2 // Needs to update if number of containers changes in deployment.
 
 	// Append upgrade guard as sidecar.
-	if _, ok := cluster.Spec.NodeManagerFeatures[upgradeGuardFeatureKey]; ok {
-		mutator := getSideCarContainerMutator(nextIndex, cluster.Spec.Images.UpgradeGuardContainer, os.Getenv(kubeUpgradeGuardEnvVar), "upgrade-guard", upgradeGuardReadinessProbeInterval)
+	if context, ok := cluster.Spec.NodeManagerFeatures[upgradeGuardFeatureKey]; ok {
```

Comment:
Is `context` the best name for this? just confused me a little bit on initial read through with actual go `context` package